### PR TITLE
Update setup.cfg

### PR DESCRIPTION
**Pull request by josiah-tesfu**
Changed the `packages` field in setup.cfg to use `find:` instead of listing each module individually. The `edgepi_rpc_server` folder has a lot of modules now so I feel this would be less error-prone.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+package_dir=
+    =edgepi_rpc_client/services
+packages=find:
+
+[options.packages.find]
+where=edgepi_rpc_client/services
+install_requires = 
+	protobuf>=3.20.3
+	edgepi-rpc-protobuf>=1.0.15
+	python-periphery>=2.4.1
```
Comment (farzadpanahi) [inline setup.cfg]: are you sure you need to include `python-periphery` ?

File: setup.cfg [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+	pyzmq>=25.1.0
+	edgepi-python-sdk>=1.2.13
+Z
```
Comment (farzadpanahi) [inline setup.cfg]: what is this last line `Z` ?

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 1 above)
Comment (josiah-tesfu) [reply setup.cfg]: Did a quick check and it's not needed. Removed.

File: setup.cfg (hunk 2 above)
Comment (josiah-tesfu) [reply setup.cfg]: Typo. Thanks for catching that.

File: requirements.txt [hunk 3]
```diff
@@ -1,5 +1,4 @@
 protobuf==3.20.3
-python-periphery==2.4.1
 pyzmq==25.1.0
 edgepi-python-sdk==1.2.8
```
Comment (farzadpanahi) [inline requirements.txt]: update sdk version

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

**Review by farzadpanahi — DISMISSED** [review]

**Review by farzadpanahi — APPROVED** [review]

**Pull request merged**